packages/front/src/components/ListDraver.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <el-drawer
+    v-model="store.state.showListDialog"
+    title="HISTmap"
```

Comment:
Use `title` slot to insert custom title DOM structure looking like: `[small world map svg] HISTmap`
https://element-plus.org/en-US/component/drawer.html#drawer-slots